model/websocket_client.go

```diff
@@ -16,17 +16,19 @@ const (
 	PING_TIMEOUT_BUFFER_SECONDS = 5
 )
 
+// WebSocketClient stores the necessary information required to
+// communicate with a websocket endpoint.
```

Comment:
Here and in several cases below we should use either `WebSocket` or `websocket`.  I would probably prefer the former since it's the official name of the protocol but the main point is that we stick with one.